### DISCUSSION TITLE
Unknown fields are not reported as error

### DISCUSSION THREAD

**Discussion by kubijo** [graphql-hive/graphql-eslint#1896] [General]
Is it an expected behavior that fields in the operation are not reported as errors?
I have validated that I do get errors when I (for example) omit a curly brace, but not for fields that clearly do not exist.

I do get an error from the IDE integration plugin: 
![image](https://github.com/B2o5T/graphql-eslint/assets/11244314/13ac2ed0-a856-420f-a8bc-76ed94a292e5)

This is my config for the relevant project

```javascript
module.exports = {
    overrides: [
        {
            files: ['*.tsx'],
            processor: '@graphql-eslint/graphql',
        },
        {
            files: ['*.graphql'],
            parser: '@graphql-eslint/eslint-plugin',
            extends: ['plugin:@graphql-eslint/schema-all'],
            rules: {
                '@graphql-eslint/alphabetize': 'off',
            },
        },
    ],
};
```

**Comment by dimaMachina** [comment]
You didn’t extend operations config, take a look in examples folder there is various examples

**Reply by kubijo** [reply]
I have been trying to find any examples of such a configuration, but the only reference to `parserOptions.operations` was here and no config example… am I just blind here? :thinking: 

![image](https://github.com/B2o5T/graphql-eslint/assets/11244314/c819257f-21a0-4a0f-8dbb-d775a801124f)

**Reply by kubijo** [reply]
Hmm, it does seem to work now just by adding `'plugin:@graphql-eslint/operations-all'` to the extends array, but it was complaining the for this, operations must be configured somehow :thinking:
